test/mesh.jl

```diff
@@ -202,6 +202,9 @@
     topo = topology(mesh)
     @test collect(faces(topo, 2)) == [connect((3,4,1), Triangle)]
     @test collect(faces(topo, 3)) == [connect((1,2,3,4), Tetrahedron)]
+
+    # use above mesh with tetrahedron for testing isinside() error
+    @test_throws ErrorException("This function only works for surface meshes with triangles as elements.") isinside(P3(0,0,0), mesh)
```

Comment:
Please more this test to be close to the other tests you introduced. If you need to create a different test mesh, that is fine. But each paragraph in our test suite is related to a specific PR usually.